packages/components/src/modal/index.tsx

```diff
@@ -85,6 +86,8 @@ function UnforwardedModal(
 	const [ hasScrolledContent, setHasScrolledContent ] = useState( false );
 	const [ hasScrollableContent, setHasScrollableContent ] = useState( false );
 
+	const contentWidthClass = contentWidth ? `width-${ contentWidth }` : '';
```

Comment:
I've noticed that additional CSS classes on components usually start with `is` or `has`. Like `is-full-screen` and `has-icon`: 

<img width="515" alt="Screenshot 2023-09-14 at 3 48 18 PM" src="https://github.com/WordPress/gutenberg/assets/35543432/6e296f09-1743-4a15-9840-8de90c347536">

I'm not sure if there's documentation about it, but it stood out to me as different from the norm. Maybe it could be some combination like `has-${ contentWidth }-width` or `is-${ contentWidth }-small` instead?

Reply:
Good observation. `has-${ contentWidth }-width` FTW!